hack/docgen/schema.go

```diff
@@ -9,18 +9,19 @@ import (
 	"fmt"
```

Comment:
Main changes in docgen:
- Make it capable of generating regular docs (`*_doc.go`) and JSON schemas independently - you can now request one or the other, or both.
- Make it able to scan go package directories recursively to generate a single JSON schema from multiple packages with config structs inside them.